day_6/exercises/student.rb

```diff
@@ -0,0 +1,16 @@
+class Student
+  attr_accessor :first_name, :last_name, :primary_phone_number
+
+  def introduction(target)
+    puts "Hi #{target}, I'm #{first_name}!"
+  end
+
+  def favorite_number
+    7
+  end
+end
+
+frank = Student.new
+frank.first_name = "Frank"
+frank.introduction("Katrina")
```

Comment:
Interesting note- since you do not have an initialize method, you are actually first defining the attribute `first_name` on the instance of your student when you call it.